units/distro_overallocated_terminations.go

```diff
@@ -109,12 +109,13 @@ func (j *hostDrawdownJob) Run(ctx context.Context) {
 
 func (j *hostDrawdownJob) checkAndTerminateHost(ctx context.Context, h *host.Host, drawdownTarget *int) error {
 
-	_, idleTime, err := j.getTimesFromHost(ctx, h)
-
-	if err != nil {
-		return errors.Wrapf(err, "error getting communication and idle time from '%s'", h.Id)
+	exitEarly, err := j.checkTerminationExemptions(ctx, h)
```

Comment:
even though you always return true for exitEarly where you return an error, that's not conventional / isn't something the code should assume, so I would still add a `err != nil` check (could fold in with exitEarly though)